commands/ngrok.cmd

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+[[ ! ${WARDEN_DIR} ]] && >&2 echo -e "\033[31mThis script is not intended to be run directly!\033[0m" && exit 1
+
+WARDEN_ENV_PATH="$(locateEnvPath)" || exit $?
+loadEnvConfig "${WARDEN_ENV_PATH}" || exit $?
+assertDockerRunning
+
+if [[ "${WARDEN_PARAMS[0]}" == "help" ]]; then
+  $WARDEN_BIN ngrok --help || exit $? && exit $?
+fi
+
+NGINX_CONTAINER=$(docker inspect -f '{{ index (split .Name "/") 1 }}' "$($WARDEN_BIN env ps -q nginx)")
+if [[ ! ${NGINX_CONTAINER} ]]; then
+    fatal "No container found for nginx service."
+fi
+
+NETWORK_NAME=$(docker container inspect ${NGINX_CONTAINER} --format '{{range $key, $value := .NetworkSettings.Networks}}{{printf "%s\n" $key}}{{end}}')
+if [[ ! ${NETWORK_NAME} ]]; then
+    fatal "No networks found for nginx service."
+fi
+
+docker run --rm -it --link ${NGINX_CONTAINER} --net ${NETWORK_NAME} -e NGROK_AUTHTOKEN=${WARDEN_NGROK_TOKEN} ngrok/ngrok:latest http ${NGINX_CONTAINER}:80
```

Comment:
I think this would be better if the `NGROK_AUTHTOKEN` was checked as a parameter first, and then fall back to `WARDEN_NGROK_TOKEN`.  This would better support one-off instantiations.

For bonus points, we could offer to save the token to the `~/.warden/.env` file - but I'm not going to ask you to do that amount of work if you don't want to.

I would then suggest that `WARDEN_NGROK_TOKEN` be set in the `~/.warden/.env` file instead of the project file - as unlike Blackfire, Ngrok authentication is more likely to be per-user rather than per-project.

Reply:
will do

Reply:
@anbis Appreciate your efforts here <3